.github/workflows/Package_pre_release.yml

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Package Pre-Release
+on:
+  push:
+    paths:
+      - 'src/**'
+      - '!src/**/*.md'
+    branches:
+      - main
+  workflow_dispatch:
+
+env:
+  PACKAGE_NAME: Suite_3_Performance_Test_Suite_Package_Pre-Release
+
+jobs:
+  source-code:
```

Comment:
Why do you need to 2 jobs here  source-code and aditional-files ?  you can do it in one job ,so that you can reduce few steps here